typings/components/PageControl.d.ts

```diff
@@ -3,7 +3,8 @@ import {GestureResponderEvent, StyleProp, ViewStyle} from 'react-native';
 import {BaseComponent} from '../commons';
 import {ColorValue} from '../style/colors';
 
-export interface PageControlProps {
+export interface PageControlProps extends JSX.IntrinsicAttributes {
+  limitShownPages: boolean;
```

Comment:
I believe this should be an optional prop

Reply:
Hm, I didn't notice that all of these are optional, sorry